src/features/auth/components/Header/index.js

```diff
@@ -2,22 +2,48 @@ import React from 'react';
 import { Trans } from 'react-i18next';
 import { Link } from 'react-router-dom';
 
-import { Header, Menu } from '../../../../common/components';
+import { Header, Icon, Menu, Popover } from '../../../../common/components';
 
 const AuthHeader = () => (
   <Header>
-    {() => [
-      <Menu.Item key="signup">
-        <Link to="/auth/sign-up">
-          <Trans i18nKey="nav.signup">Sign Up</Trans>
-        </Link>
-      </Menu.Item>,
-      <Menu.Item key="login">
-        <Link to="/auth/login">
-          <Trans i18nKey="nav.login">Log In</Trans>
-        </Link>
-      </Menu.Item>,
-    ]}
+    {({ activePathname, hide, show, isVisible, isMobile }) => {
+      const menu = (
+        <Menu
+          mode={isMobile ? 'inline' : 'horizontal'}
+          theme={isMobile ? undefined : 'dark'}
+          selectedKeys={[activePathname]}
+        >
+          <Menu.Item key="signup">
+            <Link to="/auth/sign-up">
+              <Trans i18nKey="nav.signup">Sign Up</Trans>
+            </Link>
+          </Menu.Item>
+          <Menu.Item key="login">
+            <Link to="/auth/login">
+              <Trans i18nKey="nav.login">Log In</Trans>
+            </Link>
+          </Menu.Item>
+        </Menu>
+      );
+
+      return isMobile ? (
+        <Popover
```

Comment:
lets get rid of popover

Reply:
or lets use another ant component that we dont need to write 100lines of css for